### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by farishkash** [RASL10/capstone#4]
Overall met expectations

Html is welled structured no major issues there.
Alt text should be used as the web as a large number of visually impaired visitors.

I did't see any major flaws or errors. The css was efficient and wasn't repetitive.  My main concern with your css the use of multiple media queries for the same device size.

Good organizational practices with comments in your html and css file.

Be careful with your use of flex wrap, in order to force a wrap you need to have a set width.

Since you just finished off with flexbox I like to mention css grids.
CSS grids is a new positioning feature that works with flexbox.

They will be fully supported by browsers soon.

Take a look at this game and see how far you can get to help prepare you for them.

http://cssgridgarden.com/

Also if you haven't done so already we added css grid as the last section of our css course now.

https://www.codecademy.com/learn/learn-css

Maybe as a final challenge see if you can incorporate it into this site.
